purview/api/v1.py

```diff
@@ -94,5 +94,8 @@ def get_resource_story(resource, uid):
     if len(results) == 0:
         # If no relationships are found, return the artifact by itself
         results.append([item.__label__, item.serialized])
+    else:
+        corelated_nodes = get_corelated_nodes(results, count=True)
```

Comment:
After discussion, we determined, `get_corelated_nodes` should accept `results_unordered` as the input instead of the formed JSON in `results`. This is because `results_unordered` is easier to work with and will not likely change where as the JSON being returned may change in the future.